themes/default/content/blog/announcing-v5.0.0-of-the-pulumi-aws-provider/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Announcing v5.0.0 of the Pulumi AWS Provider"
+date: 2022-03-23
+meta_desc: "v5.0.0 of the Pulumi AWS Provider is now available"
+meta_image: meta.png
+authors:
+- paul-stack
+tags:
+- aws
+---
+
+We are excited to announce v5.0.0 of the Pulumi AWS Provider. Pulumi prides itself on how it continually tries to ensure
+that major version releases are compatible with existing applications. The upstream provider that this is based upon, has had
+a major breaking change in the S3 Bucket resource. In order to ensure that our users are not impacted, We have extended
+our provider to provider both the old implementation and the new implementation of the S3 Bucket resource.
+This means that all existing usages of S3 Bucket will continue to work as normal. We also now offer the ability to
+create a v2 variants of the S3 Bucket.
```

Comment:
```suggestion
We are excited to announce v5.0.0 of the Pulumi AWS Provider. Pulumi prides itself on ensuring
that major version releases are compatible with existing applications. The upstream provider this is based upon had
a major breaking change in the S3 Bucket resource. We have extended
our provider to include both the old and new implementations of the S3 Bucket resource to prevent users from being impacted.
All existing S3 Buckets will continue to work. We also now offer the ability to
create a v2 variant of the S3 Bucket.
```